Samples/Windowing/cs-winui/ZOrder.xaml.cs

```diff
@@ -0,0 +1,130 @@
+        }
+
+        private void SwitchZOrder(object sender, RoutedEventArgs e)
+        {
+            // Bail out if we don't have an AppWindow object.
+            if (_mainAppWindow != null)
+            {
+                switch (sender.As<Button>().Name)
+                {
+                    case "BottomZOrderBtn":
+                        _mainAppWindow.MoveInZOrderAtBottom();
+                        break;
+
+                    case "TopZOrderBtn":
+                        _mainAppWindow.MoveInZOrderAtTop();
+                        break;
+
+                    default:
+                        
+                        break;
+                }
+            }
+        }
+
+        private void myNewWindowButton_Click(object sender, RoutedEventArgs e)
+        {
+            // Count up the number of windows created so far, this is just used for Title generation so we don't collide
+            m_windowCount++;
+            Window nextWindow = new Window();
+            nextWindow.Content = new TextBlock() { Text = "A wild window appears!" };
+            nextWindow.Title = "Window #" + m_windowCount.ToString();
+            nextWindow.Activate();
+            
+            // Get the WindowId for the new XAML Window and store it in a list of known "secondary" windows for the app.
+            var nextWindowHWnd = WinRT.Interop.WindowNative.GetWindowHandle(nextWindow);
+            Microsoft.UI.WindowId nextWindowId = Microsoft.UI.Win32Interop.GetWindowIdFromWindow(nextWindowHWnd);
+
+            windowList.Add(nextWindowId);
+            myListBox.Items.Add(nextWindow.Title);
+            myOtherListBox.Items.Add(nextWindow.Title);
+
+            Microsoft.UI.Windowing.AppWindow nextAppWindow = Microsoft.UI.Windowing.AppWindow.GetFromWindowId(nextWindowId);
+            nextAppWindow.Closing += NextAppWindow_Closing;
+        }
+
+        private void NextAppWindow_Closing(AppWindow sender, AppWindowClosingEventArgs args)
+        {
+            // Find the window that is closing in our list of WindowIds and remove it from
+            // that list as well as the listboxes to prevent the user from selecting a window that is no longer available.
+            int index = windowList.FindIndex(x => x == sender.Id);
+            if(index != -1)
+            {
+                windowList.RemoveAt(index);
+                myListBox.Items.RemoveAt(index);
+                myOtherListBox.Items.RemoveAt(index);
+
+                for (int i = 0; i < windowList.Count; i++)
+                {
+                    var new_title = i + 1;
+                    WindowId windowId = windowList[i];
+                    AppWindow iteratedAppWindow = AppWindow.GetFromWindowId(windowId);
+                    iteratedAppWindow.Title = "Window #" + new_title.ToString();
+                    myListBox.Items[i] = "Window #" + new_title.ToString();
+                    myOtherListBox.Items[i] = "Window #" + new_title.ToString();
+                }
```

Comment:
Why doing this part? If you are updating the titles of the windows to reflect the removal of windows you don't need to use the m_windowCount variable above.

Reply:
@emhuang-msft

Reply:
nvm, its changed

Reply:
I still see this code in ZOrderPage.xaml.xs i this PR branch.

Reply:
Oh. Never mind, looks like Emily went with the "update window title"-approach rather than the "keep a running count for new windows". 😃